microsoft-edge/webview2/concepts/process-related-events.md

```diff
@@ -0,0 +1,293 @@
+---
+title: Handling process-related events in WebView2
+description: Process-related events in WebView2 and how apps can handle them for recovery and improved reliability.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 07/11/2023
+---
+# Handling process-related events in WebView2
+
+WebView2 uses multiple processes to support the WebView2 controls in your application. Because these processes can exit during use, WebView2 provides the `CoreWebView2.ProcessFailed` and `CoreWebView2Environment.BrowserProcessExited` events for your application to react to different scenarios.  Use this document to learn how to use these events to react when these scenarios occur.
+
+To improve the reliability of your WebView2 application, it is recommended that your app handles at least the following events:
+* [The main browser process has exited unexpectedly](#the-main-browser-process-has-exited-unexpectedly).
+* [A process rendering content in the WebView2 control has exited unexpectedly](#a-process-rendering-content-in-the-webview2-control-has-exited-unexpectedly).
+* [A renderer process becomes unresponsive](#handle-unresponsive-renderers).
+
+For a list of APIs covered by this article, see [Process management](overview-features-apis.md#process-management) in _Overview of WebView2 features and APIs_.
```

Comment:
@mikehoffms should this be something like "process-related APIs"? This article has references to two kinds of API:
* Process related
* General WebView APIs (Close, Reload)

We removed the general APIs from "Process Management".

Reply:
locally, changed to:
```
To use this article, we recommend first reading [Process model for WebView2 apps](./process-model.md).  For a list of process-related APIs covered by this article, see [Process management](overview-features-apis.md#process-management) in _Overview of WebView2 features and APIs_.
```